config/keyframe_initializer.conf

```diff
@@ -0,0 +1,82 @@
+# Algorithm to use for 'initializer'.
+# Must be one of the following options:
+# 	- core
+type = keyframe
+
+
+block keyframe
+  # include the default camera intrinsics
+  block base_camera
+    include $LOCAL{default_camera_config}
+  endblock
+
+  # include bundle adjustment parameters
+  block bundle_adjuster
+    include ceres_bundle_adjuster.conf
+
+    # Maximum number of iteration of allow
+    ceres:max_num_iterations = 10
+
+    ceres:optimize_focal_length = false
+    
+    ceres:loss_function_type = SOFT_L_ONE_LOSS
+  endblock
+  
+  similarity_estimator:type = vxl
+
+  # Algorithm to use for 'camera_optimizer'.
+  # Must be one of the following options:
+  # 	- vxl
+  camera_optimizer:type = vxl
+
+  # Algorithm to use for 'essential_mat_estimator'.
+  # Must be one of the following options:
+  # 	- vxl
+  essential_mat_estimator:type = vxl
+  
+  estimate_pnp:type = ocv
+
+  # The number of samples to use in RANSAC
+  essential_mat_estimator:vxl:num_ransac_samples = 500
```

Comment:
@bclipp-kitware You dropped this to 100 in the core initializer algorithm.  I recall 100 not be enough for a robust solution but it was painfully slow at 500.  Should I change the other one back to 500?  Should I change this one to 100?  Is there a reason they should be different?